src/frontend/web_application/src/components/form/FormGrid/style.scss

```diff
@@ -31,6 +31,10 @@
 
       padding-right: map_get($co-form__spacing, 'medium');
 
+      &--full {
+        padding-right: 0;
```

Comment:
If I look only at this directive, it's sounds very weird: a right padding doesn't make a column full width itself.

If I understand right the `--full` modifier is used to fix an incorrect padding added at medium breakpoint. 
What if we just remove the padding above ?

Reply:
ok to remove "padding-right" but so, we have to add a modifier "--right-spaced" for columns that needs this right padding (most of time they need it).

Reply:
btw would it be good to set an `rightSpaced` default prop to `true`?

```
FormColumn.defaultProps = {
  className: undefined,
  size: undefined,
  fluid: false,
  bottomSpace: false,
  rightSpaced: true,
};
```

Reply:
I forgot to answer "yes"